apis/r/R/SOMAExperimentAxisQuery.R

```diff
@@ -994,6 +1132,233 @@ SOMAExperimentAxisQuery <- R6::R6Class(
       }
       return(mat)
     },
+    # Helper methods to load aspects of a measurement for the ecosystems
```

Comment:
A little more documentation here would benefit other devs because there some nuances here:

- column_names is polymorphic
- a data.frame with 0 columns is returned if column_names = FALSE
- all columns are retrieved if column_names = TRUE